MMUL-BASH-01/Exercises/Day-4/Exercise.006

```diff
@@ -0,0 +1,13 @@
+# Exercise 006 - Print every element of the array
+
+1) Using the previously defined array $tarantino, print all elements one per line
+   (remember to quote as "${tarantino[@]}")
```

Comment:
It is not clear what "to quote" stands for. Please add a ; at the end of each point.